mision_to_mars/src/mision_to_mars/Simulation.java

```diff
@@ -1,146 +1,130 @@
-
-            Item item = new Item(separated[0], Integer.parseInt(separated[1]));
-            items.add(item);
+    /**
+     * This method returns a list of items in a document
+     * path e.g "items_to_load/phase-1.txt"
+     * @return a list of items
+     */
+    public ArrayList<Item> readData(String path){
+        try {
+            return getAlistItem(path);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
         }
+        return null;
     }
 
-    public ArrayList<Rocket> loadU1(List<Item> itemsU1) {
-        System.out.println("loading U1...");
-        rocketU1 = new ArrayList<>();
-        Rocket rocket = new U1();
-        Iterator iterator = itemsU1.iterator();
-
-        while (iterator.hasNext()) {
-            Item item = (Item) iterator.next();
-            if (rocket.canCarry(item)) {
-                rocket.carry(item);
-            } else {
-                rocketU1.add(rocket);
-                rocket = new U1();
-                System.out.println("New U1 rocket created");
-                rocket.carry(item);
-            }
-            if (!iterator.hasNext()) {
-                rocketU1.add(rocket);
+    /**
+     * This method returns a list of items in a rocket
+     * @param rocket
+     * @param listOfItemsToCarry
+     * @return this method returns a list of items loaded in a rocket
+     */
+    public ArrayList<Item> loadItems(Rocket rocket, ArrayList<Item> listOfItemsToCarry){
+        System.out.println("Adding Items to "+ rocket.getClass().getTypeName());
+        ArrayList<Item> listOfItemsCarriedInARocket = new ArrayList<>();
+        for (Item item: listOfItemsToCarry) {
+            if(rocket.canCarry(item)){
+                rocket.setWeightCarried(item.getWeight());
+                listOfItemsCarriedInARocket.add(item);
             }
         }
-
-        return rocketU1;
+        return listOfItemsCarriedInARocket;
     }
 
-    public ArrayList<Rocket> loadU2(ArrayList<Item> itemsU2) {
-        System.out.println("loading U2");
-        rocketU2 = new ArrayList<>();
-        Rocket rocket = new U2();
-        Iterator iterator = itemsU2.iterator();
-
-        while (iterator.hasNext()) {
-            Item item = (Item) iterator.next();
-            if (rocket.canCarry(item)) {
-                rocket.carry(item);
-            } else {
-                rocketU2.add(rocket);
-                rocket = new U2();
-                System.out.println("New U2 rocket created");
-                rocket.carry(item);
-            }
-            if (!iterator.hasNext()) {
-                rocketU2.add(rocket);
-            }
-        }
-
-        return rocketU2;
+    /**
+     * This method run simulation for two rockets
+     */
+    public void runSimulation(){
+        //create a rockets
+        U1 U1Rocket = new U1();
+        U2 U2Rocket = new U2();
+        System.out.println("\u001B[1mAdd items to rockets for the phase 1");
```

Comment:
what is "u001B"??

Reply:
It is for a bold letter e.g.  **Add items to rockets for the phase 1**